core/src/com/unciv/ui/worldscreen/WorldScreen.kt

```diff
@@ -753,7 +751,7 @@ class WorldScreen(val gameInfo: GameInfo, val viewingCiv:CivilizationInfo) : Cam
                     .flatMap { it.cities.asSequence() }.any { viewingCiv.exploredTiles.contains(it.location) }
         }
         displayTutorial(Tutorial.ApolloProgram) { viewingCiv.hasUnique("Enables construction of Spaceship parts") }
-        displayTutorial(Tutorial.SiegeUnits) { viewingCiv.getCivUnits().any { it.type == UnitType.Siege } }
+        displayTutorial(Tutorial.SiegeUnits) { viewingCiv.getCivUnits().any { it.type!!.name == "Siege" } }
```

Comment:
Hardcoding - I'd have left the knowledge how to recognize a Siege unit in the UnitType class - `isSiege()` or a const or a factory in the companion, all would do it. With an instance comparison in the 'factory' choice, one should make sure the class satisfies the pattern for use as HashMap key - which is a good idea for this class anyway.